x/mongo/driver/operation/list_collections.go

```diff
@@ -46,6 +47,7 @@ func NewListCollections(filter bsoncore.Document) *ListCollections {
 
 // Result returns the result of executing this operation.
 func (lc *ListCollections) Result(opts driver.CursorOptions) (*driver.ListCollectionsBatchCursor, error) {
+	// opts.ServerAPIOptions = lc.serverAPI
```

Comment:
Why is this commented out?

Reply:
Shouldn't be! Uncommented. This addresses @iwysiu's comment, too.